### DISCUSSION TITLE
Lesson 11: Can't deploy NFT.

### DISCUSSION THREAD

**Discussion by realglass** [smartcontractkit/full-blockchain-solidity-course-py#1595] [Q&A] [answered]
Hi! :)

When trying to create and deploy my SimpleCollectible this error is being returned. I tried to figure it out but I am honestly a bit lost with this one. 

![image](https://user-images.githubusercontent.com/101928609/180257055-a5563448-c965-4bae-979d-ab31b9853b57.png)

Here is my SimpleCollectible.sol code:
![image](https://user-images.githubusercontent.com/101928609/180257308-382fbfa4-1691-4aba-a542-94eb6c8c917a.png)

Config code:
![image](https://user-images.githubusercontent.com/101928609/180257418-9cdb010e-1f57-4382-bf52-6391ae8d501d.png)

Deploy and create code:
![image](https://user-images.githubusercontent.com/101928609/180257504-b10e84cc-450e-4e03-b4bd-9f3f702eaf41.png)

helpful_scripts code:
![image](https://user-images.githubusercontent.com/101928609/180257739-35f5183c-5c5d-4b12-a71a-aae3d56d895a.png)

I would truly appreciate any help!! cheers!

**Comment by cromewar** [comment, marked as answer]
Hello @realglass 
you are using `from, account` with a comma, the correct form is `{"from": account}` give it a try ;)

**Reply by realglass** [reply]
oh god! thank you broo! that was the issue

**Reply by cromewar** [reply]
I'm glad is solved now!
